src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -66,12 +77,13 @@
     <MakeDir Condition="'$(CoverageEnabledForProject)'=='true'" Directories="$(CoverageReportDir)" />
 
     <Exec Command="$(TestHost) $(TestCommandLine)"
-          WorkingDirectory="$(TestPath)%(TestTargetFramework.Folder)"
+          WorkingDirectory="@(TestTargetFrameworkPath)"
```

Comment:
This "%" he is what made this target batch on all TestTargetFrameworks but by removing from the target it we are no longer batching.